protocol/v1/qbft/validation/signedmsg/msg_round.go

```diff
@@ -1,18 +1,62 @@
 package signedmsg
 
 import (
+	"bytes"
+	"fmt"
+
 	specqbft "github.com/bloxapp/ssv-spec/qbft"
-	"github.com/pkg/errors"
 
+	"github.com/bloxapp/ssv/protocol/v1/qbft"
 	"github.com/bloxapp/ssv/protocol/v1/qbft/pipelines"
 )
 
 // ValidateRound validates round
 func ValidateRound(round specqbft.Round) pipelines.SignedMessagePipeline {
 	return pipelines.WrapFunc("round", func(signedMessage *specqbft.SignedMessage) error {
 		if round != signedMessage.Message.Round {
-			return errors.Errorf("message round (%d) does not equal state round (%d)", signedMessage.Message.Round, round)
+			return fmt.Errorf("round is wrong")
```

Comment:
we need the round info in the log. 
let's map it instead or check prefix for err in tests

Reply:
@nivBlox What if we add a log with the round info (probably even outside `ValidateRound`) and still return `round is wrong`? If that doesn't sound good to you, I will map it.

Reply:
Added a log outside `ValidateRound`